app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [
+        Person(person["name"], person["age"])
+        for person in people
+    ]
+
+    Person.people = {
+        person.name: person
+        for person in person_list
+    }
```

Comment:
you can implement the logic of adding the newly created instance to the Person.people dict in the __init__ method

Reply:
def init(self, name: str, age: int) -> None:
self.name = name
self.age = age
self.people[name] = self